### PR TITLE
include sortForward in copy() of columns

### PR DESCRIPTION
**Pull request by LHjohnjanus**
include field sortForward when copying XViewerColumns to keep initial descending sort order.

### CODE REVIEW

**Review by lcaron — APPROVED** [review]
Ok for me

**Comment by wimjongman** [comment]
ok to test

**Comment by wimjongman** [comment]
John can you please also make a bugzilla for this issue. Do you want to include it in 2.2.0 which is imminent or do you want to move this to 2.3.0

**Comment by wimjongman** [comment]
add to whitelist

**Comment by LHjohnjanus** [comment]
I did add a bugzilla entry:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=549093

If you're ok with 2.2.0, I'd be thrilled.

**Comment by wimjongman** [comment]
[Bug 549093](https://bugs.eclipse.org/bugs/show_bug.cgi?id=549093) - sort order of XViewerColumn is not retained on copy

**Comment by lcaron** [comment]
retest this please

**Comment by LHjohnjanus** [comment]
Is there anything I should have done / should do to merge this pull request?

**Comment by lcaron** [comment]
John, you did everything right :)

We are waiting for an approval for the main commiter of this widget. 

I'm gonna ping him

**Comment by wimjongman** [comment]
John, can you please rebase. We are about to release.

**Comment by LHjohnjanus** [comment]
Done

**Review by wimjongman — APPROVED** [review]

**Comment by wimjongman** [comment]
Thanks, John!

**Pull request merged**
